src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(0.5),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chip: {
+      borderRadius: '4px',
```

Comment:
Let's keep an eye out. If other "chip" designs also have 4px radius then we should just move this to the theme.